packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,70 @@
+# This file contains the analysis options used by the Dart analyzer
+# to decide what errors, warnings, and hints should be shown.
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: false
```

Comment:
Sorry... more context... 

Per our discussion today which I recorded in https://github.com/flutter/flutter/issues/5488, this file will contain the strict mode options and another file will contain the non-strict mode options. That way, each can evolve as both the analyzer changes and as we learn more about what our customers need and want. In this strict mode file, I think we should have strong mode turned on similar to what we do in the `.analysis_options` in the flutter repo root. In the soon to be created non-strict options file, we will have strong mode turned off.